### PR TITLE
chore:move jump-to functionality inside namespace for shell

### LINKED ISSUE #16360: Organise Shell Context to match Figma designs
In the design file Shell covers: ACtivity Center, Jump-To, Scan and Share. 

https://www.figma.com/file/eDfxTa9IoaCMUy5cLTp0ys/Shell-for-Mobile?type=design&node-id=2545-165763&mode=design&t=NOYKKKdFVBgDZlic-0

It will help organisation of the codebase if all of these functionalities are contained in one context within `status-im2/contexts` 

i.e this structure
```
src/status-im2/contexts/shell
|-activity-center/....
|-jump-to/....  
|-scan/....    
|-share/....
```

### PR DESCRIPTION
**Pull request by J-Son89**
fixes: https://github.com/status-im/status-mobile/issues/16360

As the issue describes, the design team refers to Shell as multiple facets of our codebase. This pr is the first step to put Activity Center, Share, Jump To and Scan into one context - I will add the others in follow ups to keep the pr's as small as possible.

i.e this pr changes what is currently `shell` to `jump-to` (as the design file calls it) and then moves it inside `shell` context so other functionality can be moved there next.

This pr does not update the name for events, e.g `:shell/...` but I think this is still fine as they are events related to shell and no need to overcomplicate. 
Please let me know if I should adjust anything else.

To Test:
Nothing is added here, this is just. a refactor so a quick check of Jump To functionality should be sufficient.

### CODE REVIEW

File: src/status_im2/contexts/shell/jump_to/components/jump_to_screen/view.cljs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-(ns status-im2.contexts.shell.components.shell-screen.view
+(ns status-im2.contexts.shell.jump-to.components.jump-to-screen.view
```
Comment (J-Son89) [inline src/status_im2/contexts/shell/jump_to/components/jump_to_screen/view.cljs:1]: this one changes name a bit, from `shell-screen` to `jump-to-screen`

**Review by ilmotta — APPROVED** [review]
Good stuff @J-Son89, thanks for moving on with this refactoring and for doing it piecemeal.

**Review by Parveshdhull — APPROVED** [review]

**Comment by J-Son89** [comment]
Thanks for adding the label @churik! :)

**Review by siddarthkay — APPROVED** [review]

**Comment by pavloburykh** [comment]
@J-Son89 thanx for the PR! Tested, no issues from my side. It is ready for merge. 

However I just noticed that there are unresolved conflicts in src/status_im2/contexts/shell/components/shell_screen/view.cljs

If re-test will be needed after resolving please ping me up.

**Comment by J-Son89** [comment]
@pavloburykh I believe it is safe to merge without a retest as it was a straightforward rebase 👍

**Pull request merged**